src/state/apis/nft/selectors.ts

```diff
@@ -3,21 +3,59 @@ import { createSelector } from 'reselect'
 import type { ReduxState } from 'state/reducer'
 import { selectWalletId } from 'state/slices/common-selectors'
 
+import type { NftItemWithCollection } from './types'
+
+const selectNfts = (state: ReduxState) => state.nft.nfts.byId
+const selectNftCollections = (state: ReduxState) => state.nft.collections.byId
+
+export const selectNftItemsWithCollection = createSelector(
+  selectNfts,
+  selectNftCollections,
+  (nfts, collections): NftItemWithCollection[] => {
+    return Object.values(nfts).reduce<NftItemWithCollection[]>((acc, nft) => {
```

Comment:
Join on NFTs and collections data, we can't use portfolio for this yet even though that's what we should really do, a join of portfolio NFT to NFT to collection, see PR description